src/tasks/installers/operator.ts

```diff
@@ -361,16 +361,34 @@ export class OperatorTasks {
     {
       title: `Delete the Custom Resource of type ${CHE_CLUSTER_CRD}`,
       task: async (_ctx: any, task: any) => {
+        await kh.deleteCheCluster(flags.chenamespace)
+        do {
+          await cli.wait(20 * 60 * 1000) // default timeout in che operator to clean up cluster scope objects
```

Comment:
This line adds 2 minutes of delay in any case. I would avoid such a delay if it possible

Reply:
That's not possible... che-operator needs time to clean up created resources any way...

Reply:
It was my mistake. I changed to 20 sec